extensions/objects/object_defaults.json

```diff
@@ -47,5 +47,5 @@
       "units": []
     },
     "NumericExpression": "",
-    "PositionOfTerms": "both"
```

Comment:
Why is this deleted? This is used in MathEquationInput.

Reply:
reverted